modules/admin_manual/pages/installation/docker/index.adoc

```diff
@@ -189,13 +189,15 @@ It is the admin's responsibility to make the files persistent.
 To inspect the volumes run:
 [source,docker]
 ----
-docker volume ls | grep ownclouddockerserver
+docker volume ls
 ----
 
-To export the files as a tar archive run:
+The volume name depends on the project name which builds the first part of the volume and the name of the volume in the docker file. The naming pattern of the volume is `<COMPOSE_PROJECT_NAME>_<VOLUME_NAME>`. An environment variable for `COMPOSE_PROJECT_NAME` can be set and also be defined in a `.env` file. If not specified, the directory in which docker-compose is executed will be used as a name.
+
+To export the files of the project "owncloud-docker-server" as a tar archive run:
 [source,docker]
 ----
-docker run -v ownclouddockerserver_files:/mnt \
+docker run -v owncloud-docker-server_files:/mnt \
```

Comment:
Should be changed into the following as the name needs to found on each installation and is not fixed:

```suggestion
docker run -v <YOUR_DOCKER_VOLUME>:/mnt \
```